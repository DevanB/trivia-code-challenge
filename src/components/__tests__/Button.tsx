import {render} from '@testing-library/react'
import * as React from 'react'
import {Button} from '../Button'

describe('<Button/>', () => {
  it('should render without crashing', () => {
    const {container} = render(<Button />)
    expect(container).toMatchSnapshot()
  })
})
