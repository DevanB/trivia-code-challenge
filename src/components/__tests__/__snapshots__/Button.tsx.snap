// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button/> should render without crashing 1`] = `
<div>
  <button
    class="Button-q62e7v-0 hAohKb"
  />
</div>
`;
